### PR TITLE
fix: Minor fixes and Readme updated

### PR DESCRIPTION
**Pull request by anbublacky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,14 +42,16 @@ The install generator does the following
 * Adds a line in route for mounting.
 
 Generator that installs filter and table files to application
-
+    $ rails g spotlight_search filter orders --filters scope_name:filter_type
     $ rails g spotlight_search:filter orders --filters search:input order_status:multi_select status:select
```
Comment (anondita) [inline README.md:46]: why are we having this line bro?

Comment (anbublacky) [reply README.md:46]: First line is syntax, Second line is example

Comment (anondita) [reply README.md:46]: ohh okay

Comment (anondita) [reply README.md:46]: but it has an extra colon in it

**Pull request merged**